test/parallel/test-eventtarget.js

```diff
@@ -40,7 +41,15 @@ ok(EventTarget);
   ev.preventDefault();
   strictEqual(ev.defaultPrevented, false);
 }
-
+{
+  // No argument behavior - throw TypeError
+  throws(() => {
+    new Event();
+  }, TypeError);
```

Comment:
```suggestion
  }, {
    code: 'ERR_MISSING_ARGS',
    message: /type/
  );
```

Reply:
I'm not sure I want to specify that this is in fact an `ERR_MISSING_ARGS` (our implementation detail) rather than a `TypeError` (which is what browsers throw).

Chrome throws `Uncaught TypeError: Failed to construct 'Event': 1 argument required, but only 0 present.`.

Reply:
I see. I have no strong opinion on that.